src/main/scala/com/advancedtelematic/campaigner/data/DataType.scala

```diff
@@ -168,9 +169,22 @@ object DataType {
     val prepared, launched, finished, cancelled = Value
   }
 
+  /**
```

Comment:
AFAIU: all devices start as 'requested', which in the UI is shown as 'processed'. If it then moves to 'scheduled', the UI shows it as 'affected'. If it moves to 'rejected' it would be shown as 'not affected'.
I wonder if it could be worth to "document" that here. But on the other hand I'm reluctant, as that would couple the BE with the FE, and we might decide to display the stats in a different way in the future.

Reply:
Yeah, actually we have `affected` and `processed` scattered through campaigner too, not just the FE. So do I think it wouldn't hurt to shortly describe that here. Otherwise I'm afraid this would get confusing.